### PR TITLE
add algebraic petri model for scenario 2

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
@olynch @p-stokes

File: docs/src/scenarios/scenario2.md [hunk 1]
```diff
@@ -3,6 +3,79 @@
...
+
+@assert is_natural(Vax_aug_typed)
+
+function stratify_typed(pn1, pn2, type_system)
+  pn1′, pn2′ = [add_cross_terms(pn, type_system) for pn in [pn1, pn2]]
+  pb = pullback(pn1′, pn2′) 
+  return first(legs(pb)) ⋅ pn1′
+end
+
+seirhd_vax = stratify_typed(
```
Comment (ChrisRackauckas) [inline docs/src/scenarios/scenario2.md:68]: This is the function that's not defined.

**Comment by Vaibhavdixit02** [comment]
Should this be used instead of the SEIRHD model being used in #98 currently?

**Comment by ChrisRackauckas** [comment]
don't worry about it

**Comment by ChrisRackauckas** [comment]
Superceded by https://github.com/SciML/EasyModelAnalysis.jl/pull/101

**Pull request closed without merging**